src/libponyrt/lang/socket.c

```diff
@@ -66,9 +66,10 @@ typedef int SOCKET;
 #include <linux/udp.h>
 #endif
 #ifdef PLATFORM_IS_BSD
+#ifndef __DragonFly__
```

Comment:
Should this line be:
```
#ifdef PLATFORM_IS_DRAGONFLY
```
Which is defined in common/platform.h ?

Reply:
@winksaville : thanks, I will fix this.